apps/api/src/app/subscribers/dtos/chat-oauth-request.dto.ts

```diff
@@ -7,6 +7,10 @@ export class ChatOauthRequestDto {
   @IsMongoId()
   @IsDefined()
   environmentId: string;
+
+  @IsOptional()
```

Comment:
this one is optional in the flow so we won't make breaking changes, right?
if this is the case i think we still need to break it in a couple of versions.